mandatory/4-tax.js

```diff
@@ -5,7 +5,13 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(a) {
```

Comment:
You've used `a` as a parameter name here. But this doesn't offer insight into what the parameter represents. Remember your code is designed to be read by others so always try to convey meaning when you choose variable names - sometimes this can be tricky!